src/core/test/system/lender/supply.t.sol

```diff
@@ -19,6 +19,10 @@ import "../system.t.sol";
 
 contract SupplyTest is SystemTest {
 
+    function setUp() public {
+        baseSetup();
```

Comment:
to keep the pattern should we not create the investor user here?

Reply:
yes, already talked with @charleenfei